test/nightly/global.nightly.test.ts

```diff
@@ -0,0 +1,58 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE.md in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { type WebSiteManagementClient } from '@azure/arm-appservice';
+import { ResourceManagementClient } from '@azure/arm-resources';
+import { createAzureClient } from '@microsoft/vscode-azext-azureutils';
+import { createTestActionContext, type TestActionContext } from '@microsoft/vscode-azext-dev';
+import { type AzureSubscription } from '@microsoft/vscode-azureresources-api';
+import * as vscode from 'vscode';
+import { createSubscriptionContext, createWebSiteClient, ext, subscriptionExperience, type ISubscriptionContext } from '../../extension.bundle';
+import { longRunningTestsEnabled } from '../global.test';
+
+export let subscriptionContext: ISubscriptionContext;
+export let webSiteClient: WebSiteManagementClient;
+
+export const azcodeResourcePrefix: string = 'azc-app';
+export const resourceGroupsToDelete = new Set<string>();
```

Comment:
I don't really mind, but why did you decide to go with a set over just using an array? Feels like it just adds more overhead since you convert it into an array in the end anyway.

Reply:
Probably a preference thing, but I don't like the possibility of having duplicate resource groups listed in the array when each one only needs one delete command issued.  

My initial impression was that it complicates the delete logic later because we need to then check to see if we've already deleted the resource group since the possibility of duplicates exist when we don't use a set.  Let me know if you prefer it the other way and I can change it back though :P